protobuf/resource_type.proto

```diff
@@ -0,0 +1,10 @@
+syntax = "proto3";
+
+package Trento.Checks.V1;
+option go_package = "/;events";
+
+enum ResourceType {
+    UNKNOWN = 0;
```

Comment:
My bad today, I said unknown but if we want to stick to the convention, use unspecified https://protobuf.dev/programming-guides/dos-donts/#do-include-an-unspecified-value-in-an-enum